### ISSUE TITLE
raises permitted-contents error only in `.map` syntax (with JSX parser)

### ISSUE THREAD

**Issue by tnyo43** [markuplint/markuplint#2302]
- Markuplint version: `4.11.4`
- Parser lang: jsx
- Node.js version: `22.11.0`
- OS: macOS

## Describe the bug

It raises errors when I use `<path>` element in `<svg>` element with `.map` syntax in JSX.
I believe this is a valid syntax and it should not raises any error.

```jsx
<svg>
  {list.map(item => (
    <path /> {/** The "path" element is not allowed in the "svg" element in this contextMarkuplint permitted-contents */}
  ))}
</svg>
```

## Code Example or Playground URL

[Playground URL](https://playground.markuplint.dev/#N4Igxg9gdgZglgcxALhMAOlABF9ICmAHgC75QAmAznslgNp4C2AhgE4DWArgA4A2cUYslb5IjRmXL5yAWhHMwxPAF0ANJhx5ubSvlY0sGbDlwh059ADoAVpUIASA3gACLDj36CA9LcIztrLr6ZsYAvurGeJTcotQohhomeBZWvo7xLm5cfALEXvKKMtGiyWGYoZggquAQUgBicLz4ACoAnjEoIDZ2VTVSnUTcEKzEWJBQlKMAklAAbsz85AAKzMQAFjMAygBqAOJYALxYABTAWPyTqlhw5FihAJSHAHwJxiLEnKzYx4k4ADyUWYIJ6-EzAC7ESwsbjHOCkRjPE6gkxYP7adZYLwg4wo+73Co41FeQHAxL3ADc5UpUEwg2Go3GkywM3mi12Wz2hxOZwhVxud0eBxeRhw70+31BAKB2JROHBcEmUOYMLh+ARQqRhJRfwQmJlsrxBO1xOlZOpBNphCGIzG0CZ2wWN3ZUB2+yOp3OCuIfNuD0RIqwYq+mu1JP12t1WMlJtJxgpVMwlutDLtowdixW6w5bu5nsu119guFiSDEsJUtjstR6LWeujYbNCZpUDpNsZacd5AAMnAZgBVXhcj28gsC-0l-AfYM-cucXjhsEQpUq+GImdV-78OtarCG6Nz-XxqChakgUJAA)

I added some other examples (`ValidGInSVG`, `ValidPathInSVG` and `ValidLiInUl`) to compare the behaviours with the similar cases.

## Steps To Reproduce

1. open playground

## The current behavior

raises an error `error: The "path" element is not allowed in the "svg" element in this context (permitted-contents)`

## The expected behavior

should not raise any error.

**Comment by YusukeHirao** [comment]
Merged and will be available in v5.
